packages/office-ui-fabric-react/src/components/Spinner/Spinner.types.ts

```diff
@@ -78,6 +84,20 @@ export enum SpinnerSize {
   large = 3
 }
 
+export enum SpinnerLabelPosition {
```

Comment:
I know we are moving away from using `const enum`. What about `enum`? Should I make this a `const` instead?

Reply:
Can we use a string union type here to remove the need for an enum? Example:

```ts
export type SpinnerLabelPosition = 'top' | 'right' | 'bottom' | 'left';
```

cc: @kenotron

Reply:
I know @kenotron did replace `const enums` but I found that we are still using `enums` and I thought it's OK to use them. There are 2 more enums in the `Spinner.types.ts` file. What should I do regarding them to be consistent?

Reply:
enums are "okay", but we should really get rid of them and use string union types like @KevinTCoughlin suggested there.

Reply:
Wouldn't changing the current enums to string union types break everyone? I see that with the new ones we add the string union type, but with the existing ones, this would not work. Unless I am not understanding enums correctly.

Reply:
My understanding is:

For _existing_ enums, I would follow what was done in https://github.com/OfficeDev/office-ui-fabric-react/pull/7098/

But for _new_ enums, I would prefer string union types.